roles/scala/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: create scala group
+  group: 
+    name: scala
+    state: present
+
+- name: create /var/lib/scala (scala)
+  file:
+    path: /var/lib/scala
+    group: scala
+    recurse: yes
+    mode: "g+rwx"
+    state: directory
+
+- name: add oracle repo (scala)
```

Comment:
Do we need oracle java? I forgot what was running on the old prl3, but I had an impression that it was openjdk

Reply:
Nope, we used oracle java